modules/yarn.lock

```diff
@@ -4348,17 +4348,6 @@ babel-plugin-minify-type-constructors@^0.4.3:
   dependencies:
     babel-helper-is-void-0 "^0.4.3"
 
-babel-plugin-module-resolver@^3.2.0:
```

Comment:
I guess something got merged without properly updating the lockfile. So including this cleanup (didn't even need an `npx yarn-deduplicate yarn.lock`; just the `yarn add alloyeditor@2.11.14` was enough to clean out this cruft).